resources/models/resource.py

```diff
@@ -203,6 +203,8 @@ class Resource(ModifiableModel, AutoIdentifiedModel):
                                         default=ACCESS_CODE_TYPE_NONE)
     reservable_days_in_advance = models.PositiveSmallIntegerField(verbose_name=_('Reservable days in advance'),
                                                                   null=True, blank=True)
+    reservable_delay_days = models.PositiveSmallIntegerField(verbose_name=_('Days until reservation is possible'),
```

Comment:
We can't break the API backwards compatibility, but we could do the rename here, and expose the value both as `reservable_days_in_advance` and `reservable_max_days_in_advance` in the API, and mark the old one as deprecated in the Swagger docs. Then it can be removed sometime in the future when for example a new version of the API is created.

Reply:
Yeah, that sounds reasonable!

Reply:
This change made in 5808b69.